tools/c7n_gcp/tests/test_loadbalancer.py

```diff
@@ -155,6 +155,34 @@ def test_loadbalancer_ssl_policy_get(self):
         self.assertEqual(instance['kind'], 'compute#sslPolicy')
         self.assertEqual(instance['name'], 'newpolicy')
 
+    def test_loadbalancer_ssl_policy_delete(self):
+        project_id = 'custodian-test-project-0'
+        session_factory = self.replay_flight_data('lb-ssl-policy-delete',
+                                                  project_id=project_id)
+        base_policy = {'name': 'lb-ssl-policy-delete',
+                       'resource': 'gcp.loadbalancer-ssl-policy'}
+
+        policy = self.load_policy(
+            dict(base_policy,
+                 filters=[{'type': 'value',
+                           'key': 'minTlsVersion',
+                           'op': 'ne',
+                           'value': 'TLS_1_2'}],
+                 actions=[{'type': 'delete'}]),
+            session_factory=session_factory)
+        resources = policy.run()
+        self.assertEqual(2, len(resources))
+        self.assertIsNot('TLS_1_2', resources[0]['minTlsVersion'])
+        self.assertIsNot('TLS_1_2', resources[1]['minTlsVersion'])
+
+        if self.recording:
+            sleep(1)
+
+        policy = self.load_policy(base_policy, session_factory=session_factory)
```

Comment:
don't use policies to verify policies, use a client api call.

Reply:
Updated